CONTRIBUTING.md

```diff
@@ -40,41 +46,39 @@ When running `make run`, the webhooks aren't effective as it starts the manager
 1. configure a proxy between the Kubernetes API server and your host, so that it can contact the webhook in your local machine
 1. create the TLS certificates and place them, by default, on `/tmp/k8s-webhook-server/serving-certs/tls.crt`. The Kubernetes API server has also to be configured to trust the CA used to generate those certs.
 
-In general, it's just easier to deploy the manager in a Kubernetes cluster instead. For that, you'll need the `cert-manager` installed. You can install it by running:
+In general, it's just easier to deploy the operator in a Kubernetes cluster instead. For that, you'll need the `cert-manager` installed. You can install it by running:
 
 ```bash
 make cert-manager
 ```
 
-In pursuit of continuous improvement, a variable named `CERTMANAGER_VERSION` which can be run:
+Variable `CERTMANAGER_VERSION` can be used to override the cert manager version:
 ```bash
 CERTMANAGER_VERSION=1.60 make cert-manager
 ```
 
-By default, it will generate an image following the format `quay.io/${USER}/opentelemetry-operator:${VERSION}`. You can set the following env vars in front of the `make` command to override parts or the entirety of the image:
+Now let's deploy OpenTelemetry operator into the cluster. By default `make deploy` uses image with the following format `ghcr.io/${USER}/opentelemetry-operator` which can be overridden by:
```

Comment:
```suggestion
When deploying the operator into the cluster using `make deploy`, an image in the format `ghcr.io/${USER}/opentelemetry-operator` is generated. If this format isn't suitable, it can be overridden by:
```